server/src/main/java/org/apache/iotdb/db/metadata/MGraph.java

```diff
@@ -248,6 +248,10 @@ public Metadata getMetadata() throws PathErrorException {
     return mtree.getAllStorageGroup();
   }
 
+  HashSet<String> getAllDevices() {
```

Comment:
I note that all its callers Interprete the returned value as `Set` but not `HashSet`.